lib/reek/cli/utilities.rb

```diff
@@ -0,0 +1,12 @@
+module Reek
+  module CLI
+    # CLI utilities
+    module Utilities
```

Comment:
Looks my per-commit notes don't show up here, so I'll quote myself:

> This name suggests this will be a grab bag of vaguely related methods. How about calling it SilentRequirer or something like that and give it only the one purpose?